### ISSUE TITLE
not working in project with tailwindcss config in subdir

### ISSUE THREAD

**Issue by merrickluo** [tailwindlabs/tailwindcss-intellisense#668]
**What version of Tailwind CSS IntelliSense are you using?**

v0.9.1

**What version of Tailwind CSS are you using?**

v3.2.2

**What package manager are you using?**

npm

**What operating system are you using?**

Linux

**Tailwind config**

N/A

**VS Code settings**

    "tailwindCSS.includeLanguages": {
        "html-eex": "html"
    },

**Reproduction URL**

**Describe your issue**
Hi, I recently find that tailwindcss-intellisence does not work in phoenix projects that use tailwindcss in the eex/leex/heex templates. I did some debugging and found that the language server didn't treat the template files as in the project. 
the folder structure for a phoenix project is like this:

```
project
  - assets
    - tailwindcss.config.js
  - lib
    - project_web
      - live
        - template.html.heex
      - templates
        - view_name
          - view.html.eex
  ....
```
in the log:
```
  "message": "[Global] Creating projects: [{\"folder\":\"/home/xxx\",\"configPath\":\"/home/xxx/assets/tailwind.config.js\",\"isUserConfigured\":false,\"documentSelector\":[{\"pattern\":\"/home/xxx/assets/tailwind.config.js\",\"priority\":0},{\"pattern\":\"/home/xxx/assets/**\",\"priority\":3}]}]"
```
it seems only look up to the `assets/**` hence all the template files got ignored.
do you think if it's ok to add a documentselector directly to root?

**Comment by bradlc** [comment]
Hi @merrickluo. Please can you share a GitHub repository that I can open to reproduce the issue in VS Code?

**Comment by merrickluo** [comment]
Thanks for the reply, I tried to reproduce with a new project, but it works in new project
new:
![image](https://user-images.githubusercontent.com/1495875/203187318-26b69b81-430c-4e00-9cac-0dc3db096131.png)
old:
![image](https://user-images.githubusercontent.com/1495875/203187473-532baa7a-1f75-49ba-aeca-4e955b3fa4c1.png)

somehow my project is missing the project root selector with priority 4. I don't recall any obvious differences between those two projects regarding the tailwindcss setup, I'll try debug more with the old project and report back, thanks again.

**Comment by merrickluo** [comment]
Hi again @bradlc. I find out the difference now. phoenix now uses esbuild by default, the `assets/package.json` file is only created when adding npm packages. in [getPackageRoot](https://github.com/tailwindlabs/tailwindcss-intellisense/blob/6b3e598e5378812b42db8a208db4980c82b60a10/packages/tailwindcss-language-server/src/server.ts#L1505) treat the folder contains `package.json` the `rootDirectory`.

here is an example project: https://github.com/merrickluo/phoenixtw thanks again for taking a look!

**Comment by bradlc** [comment]
Hey @merrickluo, this should be fixed in `v0.9.7` 👍 The root selector will be added if there's exactly one active "project"

**Comment by merrickluo** [comment]
thanks!
